tests/test_scalars.py

```diff
@@ -78,27 +95,37 @@ def test_period() -> None:
     as4 = int(1)
     as5 = pd.period_range("2012-1-1", periods=10, freq="D")
     as6 = pd.Period("2012-1-1", freq="D")
+    as7 = cast("TimedeltaSeries", pd.Series([pd.Timedelta(days=1)]))
+    as8 = cast("PeriodSeries", pd.Series([as6]))
```

Comment:
You could avoid the `cast` here by using `pd.Series(pd.timedelta_range(...))` and `pd.Series(pd.period_range(...))` because those will return `TimedeltaSeries` and `PeriodSeries` respectively

Reply:
That is helpful. Thanks.

Reply:
Did not work for TimedeltaSeries, but did for PeriodSeries.